wgpu/src/util/belt.rs

```diff
@@ -51,11 +57,55 @@ impl StagingBelt {
     /// * 1-4 times less than the total amount of data uploaded per submission
     ///   (per [`StagingBelt::finish()`]); and
     /// * bigger is better, within these bounds.
+    ///
+    /// The buffers returned by this staging belt will have the buffer usages
+    /// [`MAP_READ | MAP_WRITE`](BufferUsages).
     pub fn new(device: Device, chunk_size: BufferAddress) -> Self {
+        Self::new_with_buffer_usages(device, chunk_size, BufferUsages::COPY_SRC)
+    }
+
+    /// Create a new staging belt.
+    ///
+    /// The `chunk_size` is the unit of internal buffer allocation; writes will be
+    /// sub-allocated within each chunk. Therefore, for optimal use of memory, the
+    /// chunk size should be:
+    ///
+    /// * larger than the largest single [`StagingBelt::write_buffer()`] operation;
+    /// * 1-4 times less than the total amount of data uploaded per submission
+    ///   (per [`StagingBelt::finish()`]); and
+    /// * bigger is better, within these bounds.
+    ///
+    /// `buffer_usages` specifies with which [`BufferUsages`] the staging buffers
+    /// will be created. [`MAP_WRITE`](BufferUsages::MAP_WRITE) will be added
```

Comment:
```suggestion
    /// `buffer_usages` specifies the [`BufferUsages`] the staging buffers
    /// will be created with. [`MAP_WRITE`](BufferUsages::MAP_WRITE) will be added
```

[This is something up with which I will not put](https://quoteinvestigator.com/2012/07/04/churchill-preposition/) 😆

Reply:
Now the only remaining uses of "with which" are not from me :stuck_out_tongue_closed_eyes: